src/pint/models/frequency_dependent.py

```diff
@@ -8,6 +8,10 @@
 from pint.models.timing_model import DelayComponent, MissingParameter
 
 
+def _description_frequency_dependent_coefficients(x):
```

Comment:
You can put this as a `classmethod` in the class `FD`, which would allow just naming it `_description_template` or whatever. I realize this doesn't fully cover all the naming complexities but it might help; also help make it clear the restricted scope/utility of these functions.